projects/ion/src/lib/dropdown/dropdown.component.ts

```diff
@@ -38,6 +38,7 @@ export class DropdownComponent implements AfterViewInit {
   @Output() searchChange = new EventEmitter<string>();
 
   iconSize = 16;
+  clearFiltersSelected = false;
```

Comment:
Transformar variável em função para testar se há algum item selecionado
```suggestion
  clearFiltersSelected(): boolean {
    const optionSelected = [];
    this.options.forEach((item: DropdownItem) =>
      item.selected === true ? optionSelected.push(item.selected) : ''
    );
    if (optionSelected && optionSelected.length > 0) return true;
    return false;
  }
```